File: 30.07.2024/prime.py
def prime(a):
    for i in range(2,a):
        if a%i!=0:
            f=1
        else:
            f=0
    if f==1:
        print("prime")
    else:
        print("not prime")

a=int(input("Enter the number : "))
prime(a)

File: 31.07.2024/prime.c
#include<stdio.h>
int main()
{
    int a,f;
    printf("Enter the number : ");
    scanf("%d",&a);
    if(a==2)
    {
        printf("Its prime :)");
    }
    for(int i=2;i<=a/2;i++)
    {
        if (a % i ==0)
        {
            f=0;
            printf("Its not prime :(");
            break;
        }
        else{
            f=1;
        }
    }
    if(f==1)
    {
        printf("Its a prime :)");
    }
}
